.github/workflows/release.yaml

```diff
@@ -32,11 +32,48 @@ jobs:
           context: .
           push: true
           tags: ngrok/ngrok-ingress-controller:latest
-    
-  helm_release:
+
+  helm_bump_version:
     needs: docker_release
     runs-on: ubuntu-latest
 
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v3
+
+      - name: Configure Git
+        run: |
+          git config user.name "$GITHUB_ACTOR"
+          git config user.email "$GITHUB_ACTOR@users.noreply.github.com"
+
+      - name: Read Helm Chart Version
+        id: helm_chart
+        uses: jacobtomlinson/gha-read-helm-chart@master
+        with:
+          path: helm/ingress-controller
+
+      - name: Increment Semver Patch-level
+        id: next_version
+        uses: "WyriHaximus/github-action-next-semvers@v1"
+        with:
+          version: "${{ steps.helm_chart.outputs.version }}"
+
+      - name: Update Chart version
+        uses: jacobtomlinson/gha-find-replace@v2
+        with:
+          include: "helm/ingress-controller/Chart.yaml"
+          find: '(?m)^version: .*$'
+          replace: "version: ${{ steps.next_version.outputs.patch }}"
+
+      - name: Commit Chart.yaml version bump
```

Comment:
Won't this commit cause another invocation of the github action and make a cyclical loop? 
I didn't realize we couldn't push over the same version, but i guess it wouldn't be too hard to just include a bump in that version as a part of the pr or as a separate pr? That could be annoying though with merge conflicts I guess 🤔 . I guess helm just doesn't really have the same `:latest` concept as docker. 

If it doesn't make a loop, this seems good for now. It will let us easily test the latest constantly, and later on I think russ and alan probably want to unify the release process for the open source repos.